test/unit/selector.js

```diff
@@ -160,6 +160,7 @@ QUnit.test( "broken", function( assert ) {
 	broken( "Broken Selector", "," );
 	broken( "Broken Selector", ",a" );
 	broken( "Broken Selector", "a," );
+	broken( "Identifer with bad escape", "foo\\\fbaz" );
```

Comment:
There's a typo here:
```suggestion
	broken( "Identifier with bad escape", "foo\\\fbaz" );
```